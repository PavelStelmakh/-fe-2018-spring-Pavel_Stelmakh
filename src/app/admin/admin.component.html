<app-user-details [role]="'admin'"></app-user-details>
