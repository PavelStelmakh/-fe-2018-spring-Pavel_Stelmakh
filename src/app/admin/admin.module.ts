import { NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';
import { Routes, RouterModule } from "@angular/router";
import { AdminComponent } from './admin.component';
import { UserDetailsModule } from '../user-details/user-details.module';
import { AuthGuard } from '../auth/auth.guard';

const route: Routes = [
  {
    path: '',
    component: AdminComponent,
    canActivate: [AuthGuard]
  }
];

@NgModule({
  imports: [
    CommonModule,
    RouterModule.forChild(route),
    UserDetailsModule
  ],
  declarations: [AdminComponent]
})
export class AdminModule { }
