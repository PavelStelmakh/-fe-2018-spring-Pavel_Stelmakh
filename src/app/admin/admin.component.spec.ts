import { TestBed, async, ComponentFixture } from '@angular/core/testing';
import { NO_ERRORS_SCHEMA } from '@angular/core';
import { AdminComponent } from './admin.component';

describe('AdminComponent', () => {
  let fixture: ComponentFixture<AdminComponent>;

  beforeEach(async(() => {
    TestBed.configureTestingModule({
      declarations: [
        AdminComponent
      ],
      schemas: [
       NO_ERRORS_SCHEMA
      ]
    }).compileComponents();

  }));

  beforeEach(() => {
    fixture = TestBed.createComponent(AdminComponent);
  });

  it('renders markup to snapshot', () => {
    fixture.detectChanges();

    expect(fixture).toMatchSnapshot();
  });

});
