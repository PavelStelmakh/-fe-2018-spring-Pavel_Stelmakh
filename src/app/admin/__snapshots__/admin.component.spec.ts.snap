// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdminComponent renders markup to snapshot 1`] = `
<app-admin>
  <app-user-details />
</app-admin>
`;
